src/commands/math/commands.js

```diff
@@ -429,6 +429,36 @@ CharCmds['/'] = P(Fraction, function(_, super_) {
   };
 });
 
+LatexCmds.underset = P(MathCommand, function(_, super_) {
+  _.ctrlSeq = '\\underset';
+  _.htmlTemplate =
+      '<span class="mq-underset mq-overunder mq-non-leaf">'
+    +   '<span class="mq-over">&1</span>'
+    +   '<span class="mq-under">&0</span>'
+    + '</span>'
+  ;
+  _.textTemplate = ['[', '|', ']'];
+  _.finalizeTree = function() {
+    this.downInto = this.ends[L].upOutOf = this.ends[R];
+    this.upInto = this.ends[R].downOutOf = this.ends[L];
+  };
+});
+
+LatexCmds.overset = P(MathCommand, function(_, super_) {
+  _.ctrlSeq = '\\overset';
+  _.htmlTemplate =
+      '<span class="mq-overset mq-overunder mq-non-leaf">'
+    +   '<span class="mq-over">&0</span>'
+    +   '<span class="mq-under">&1</span>'
+    + '</span>'
+  ;
+  _.textTemplate = ['[', '|', ']'];
```

Comment:
there's no need to change this because we don't really care about .text(), but out of curiosity was there any particular reason you chose this?